qcodes/tests/drivers/test_ami430_pyvisasim.py

```diff
@@ -0,0 +1,413 @@
+import io
```

Comment:
Nicely done! Except, I am wondering if the test file should not be called test_ami430 without the "_pyvisasim". This is just the new testing module.

Reply:
I was hoping you would say that! 😺